_codux/ui-kit/components/components.board.tsx

```diff
@@ -41,21 +49,26 @@ export default createBoard({
                         ]}
                         className={styles.accordion}
                     />
-                    <hr className={styles.hrLight} />
                 </Variant>
+                <hr className={styles.hrLight} />
                 <p className={classNames(styles.variantName, styles.headlinesSpacing)}></p>
                 <h4 className={styles.sectionHeader}>LABELS</h4>
-                <h4>Labels missing</h4>
+                <div className="relativeContainer">
+                    <Variant name="Label">
+                        <span className={productCardStyles.ribbon}>I&apos;M A LABEL</span>
+                    </Variant>
+                </div>
```

Comment:
This is not a reusable class, it's an internal part of the product card component, don't put it here.

Reply:
updated